src/hotspot/cpu/riscv/macroAssembler_riscv.cpp

```diff
@@ -4882,7 +4927,7 @@ void MacroAssembler::get_thread(Register thread) {
                       RegSet::range(x28, x31) + ra - thread;
   push_reg(saved_regs, sp);
 
-  mv(t1, CAST_FROM_FN_PTR(address, Thread::current));
+  movptr(t1, ExternalAddress(CAST_FROM_FN_PTR(address, Thread::current)));
```

Comment:
It will be more consistent with other places where we move an ExternalAddress into a register if we do:
 `la(t1, ExternalAddress(CAST_FROM_FN_PTR(address, Thread::current)))`.